tests/Microsoft.Bot.Configuration.Tests/ConfigurationLoadAndSaveTests.cs

```diff
@@ -13,7 +13,7 @@ public class ConfingurationLoadAndSaveTests
         [TestMethod]
         public async Task DeserializeBotFile()
         {
-            var config = await BotConfiguration.LoadAsync(@"..\..\test.bot");
+            var config = await BotConfiguration.LoadAsync(@"..\..\..\test.bot");
```

Comment:
There sure is a log of "...\..\..\test.bot". 

Can you make a constant and use everywhere? Const string testBotFileName="..."

Reply:
Done!